pkg/commands/logging/s3/create.go

```diff
@@ -216,6 +214,30 @@ func (c *CreateCommand) ConstructInput(serviceID string, serviceVersion int) (*f
 	return &input, nil
 }
 
+func ValidateRedundancy(val string) (redundancy fastly.S3Redundancy, err error) {
```

Comment:
As noted by the linter, can we add a comment...
```suggestion
// ValidateRedundancy identifies the given redundancy type.
func ValidateRedundancy(val string) (redundancy fastly.S3Redundancy, err error) {
```

Reply:
Thanks, added it in https://github.com/fastly/cli/pull/641/commits/7ad861493108308ff40c5d3cc4b5f6ac59c0c536